Dockerfile

```diff
@@ -22,7 +42,11 @@ RUN apt-get update &&\
         &&\
     apt-get clean
 
-WORKDIR "/noseyparker"
+COPY --from=planner /noseyparker/recipe.json recipe.json
+
+# Build dependencies - this is the caching Docker layer
+# Arguments match arguments specified in `create-release.zsh` script
```

Comment:
Please also make a note in `create-release.zsh` near the `cargo build` step that if the invocation is changed, the Dockerfile should also be updated

Reply:
Done!